vars/saveCache.groovy

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/groovy
+
+def call(String cache_file, String directory, Integer clean_old = 0, String exclude="") {
+    try {
+        if (exclude.length() > 0) {
+            sh """
+            if [ -f ${cache_file}.exclude ]; then
+                if [ "\$(cat ${cache_file}.exclude )" != "${exclude}" ]; then
```

Comment:
Looks dangerous, what if someone tries to exclude `foo/*`? Why don't we compare checksums or save the hash of the `exclude` string instead of using `cat`?